bundle/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,375 @@
+          - pods/log
+          - nodes
+          - replicationcontrollers
+          - secrets
+          - ingressclasses
+          - routes
+          - network
+          - networks
+          verbs:
+          - get
+          - watch
+          - list
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - get
+          - watch
+          - list
+        - apiGroups:
+          - ""
+          - extensions
+          - networking.k8s.io
+          resources:
+          - namespaces
+          - ingresses
+          - services
+          - pods
+          - networkpolicies
+          - routes
+          verbs:
+          - get
+          - watch
+          - list
+          - update
+          - patch
+        - apiGroups:
+          - nsx.vmware.com
+          resources:
+          - nsxerrors
+          - nsxlocks
+          - ncpconfigs
+          verbs:
+          - create
+          - get
+          - list
+          - patch
+          - delete
+          - update
+        - apiGroups:
+          - ""
+          - extensions
+          - networking.k8s.io
+          resources:
+          - ingresses/status
+          - services/status
+          - routes/status
+          verbs:
+          - replace
+          - update
+          - patch
+        - apiGroups:
+          - config.openshift.io
+          resources:
+          - networks
+          verbs:
+          - patch
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          - routes/status
+          verbs:
+          - patch
+          - update
+        - apiGroups:
+          - k8s.cni.cncf.io
+          resources:
+          - network-attachment-definitions
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - vmware.com
+          resources:
+          - loadbalancers
+          - loadbalancers/status
+          - nsxlbmonitors
+          - nsxlbmonitors/status
+          - virtualnetworkinterfaces
+          - virtualnetworkinterfaces/status
+          - virtualnetworks
+          - virtualnetworks/status
+          verbs:
+          - create
+          - get
+          - list
+          - patch
+          - update
+          - watch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - pods/exec
+          verbs:
+          - get
+          - create
+        serviceAccountName: nsx-ncp-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - networking
+  - security
+  maintainers:
+  - email: sorlando@vmware.com
+    name: Salvatore Orlando
+  - email: simeonovy@vmware.com
+    name: Yasen Simeonov
+  maturity: alpha
+  provider:
+    name: VMware
+  version: 0.3.0
```

Comment:
we might consider updating CSV version too

Reply:
> we might consider updating CSV version too

Already update to 3.2.0.